catalog/ui/search-ui/search-endpoint/src/test/java/org/codice/ddf/ui/searchui/query/controller/SearchControllerTest.java

```diff
@@ -273,21 +353,21 @@ private Query getQueryRequest(String cql) throws CQLException {
     public void testExecuteQueryCacheEnabledWithSingleSource() throws Exception {
         Set<String> srcIds = new HashSet<>(1);
         srcIds.add("id");
-        List<String> modes = cacheQuery(srcIds, 2);
+        List<String> modes = cacheQuery(srcIds, 3);
 
-        assertThat(modes.size(), is(2));
-        assertThat(modes, hasItems("cache", "update"));
+        assertThat(modes.size(), is(3));
```

Comment:
This class is already 800+ lines, I would really like to avoid creating these variables unless they really helps illuminate some facet of the test. Thankfully with IntelliJ, the method call shows the signature's name for each variable and helps the readability inside the IDE.